helm/ffc-demo-payment-service/jenkins-aws.yaml

```diff
@@ -0,0 +1,7 @@
+name: aws-ffc-demo
```

Comment:
This is overridden by the deploy command, but I think it's still cleaner to set this to the name of the service.

Reply:
Agreed.  I misread the original command and thought it appended to this value as opposed to overwriting.  Fixed.